ROADMAP.md

```diff
@@ -0,0 +1,90 @@
+# Roadmap
+
+## Overview
+
+At Ratify, our mission is to safeguard the container supply chain by ratifying trustworthy and compliant artifacts. We achieve this through a robust verification engine that seamlessly integrates with Kubernetes, container runtime and CI/CD pipelines. This engine validates supply chain metadata associated with the artifacts, including signatures and attestations (such as vulnerability reports, SBOM, provenance data, and VEX documents). Each metadata provides a unique perspective, enabling us to answer essential questions: Is this artifact unmodified since its creation? Is it free from known vulnerabilities? What are its dependencies? Who produced it, from where, and how trustworthy is the source? Additionally, Ratify offers a flexible, pluggable framework, allowing users to seamlessly integrate custom verification plugins tailored to their specific needs.
```

Comment:
Do you think we should emphasize (maybe move to the first line) the extensible aspects of Ratify even more? Ratify was designed from the beginning to be able to evolve as the CSSC landscape evolves. New verifiers to support new artifacts as it comes along. Support new policy engines/languages as necessary. And support more registries and artifact stores as needed. I feel the "flexible, pluggable framework" is one of if not the biggest differentiator for Ratify?